app/src/main/java/com/x8bit/bitwarden/ui/platform/components/stepper/BitwardenStepper.kt

```diff
@@ -63,7 +63,7 @@ fun BitwardenStepper(
                         onValueChange(decrementedValue)
                     }
                 },
-                isEnabled = isDecrementEnabled,
+                isEnabled = isDecrementEnabled && value?.let { (it - 1) in range } != false,
```

Comment:
Two questions:
- Why `value` vs `clampedValue`?
- Would you be interested in making some booleans for this like `isAtRangeMaximum` and `isAtRangeMinimum`? I just find this takes a bit of time to understand and those variable names would make it immediately obvious.